Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+def collection_path = 'ansible_collections/rudder/rudder'
+// uid of the jenkins user of the docker runners
+def user_id = "1007"
+
+pipeline {
+  agent none
+  stages {
+    stage ('typos') {
+        agent {
+            dockerfile {
+                filename 'ci/typos.Dockerfile'
+                additionalBuildArgs  '--build-arg VERSION=1.0'
+            }
+        }
+        steps {
+            sh script: './qa-test --typos', label: 'check collection typos'
+        }
+    }
+    stage ('ansible sanity tests') {
+        agent {
+            dockerfile {
+                filename 'ci/ansible-test.Dockerfile'
+                additionalBuildArgs  '--build-arg USER_ID='+user_id
+            }
+        }
+        steps {
+            sh "mkdir -p ${collection_path}"
```

Comment:
why this move?

Reply:
ansible-test requires the  collection to be in a specific directory tree: `ansible_collections/<namespace>/<collection name>` otherwise it can not start.
I could not find a way to mount the workspace in such way.